### DISCUSSION TITLE
Terminal not working when connecting to remote ssh in vscode

### DISCUSSION THREAD

**Discussion by ivorsmorenburg** [blinksh/blink#1591] [Support]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

**Comment by carloscabanero** [comment]
This is by design. When you connect to a remote project over ssh, basically using `code <host>:<path>`, the terminal is not available. This is because the VS Code we use on this instance is VS Code for Web, which does not support a hook for a terminal. For this scenario, it is more convenient to just use Blink as the terminal.

**Reply by ivorsmorenburg** [reply]
Sorry I been readi g the whole documentation and I don't find or I have missed the part where I can ssh and run terminal using vscode.

My problem it's that I have a vscode environment setup in the cloud and I was expecting to connect using ssh and hook the terminal.

Is there a way around this?
It's plan to implement the support for terminal? 

It will be a life changing feature.

**Reply by carloscabanero** [reply]
Not sure if I fully understand what you are trying to do. More details on your general setup and what you mean by "in the cloud" (where? is that yours or GitHub?), and what you have tried already, how you are currently connecting, etc... would definitely help. 

Some tips. If you have a code-server environment in the cloud, then you can connect by using `code <url>`. You should have access to the terminal inside Code. If you also want Blink to access that terminal, you have to install your public key in that machine and connect normally as with any other system.

Hope this helps.

**Reply by ivorsmorenburg** [reply]
I have a VM machine running in a Cloud provider (In the Cloud), I connect using ssh using my VScode Desktop (remote host plugin).
It will be amazing to do the same from blink.sh as web its pretty limited.

I have tried to connect to an ssh host using code <host> but Blink wont have terminal as its web based.

I think Im missing something.. if you say its possible to get the terminal running using code <url>
